src/Utilities/Compiler/Options/MSBuildPropertyOptionNames.cs

```diff
@@ -5,9 +5,9 @@ namespace Analyzer.Utilities
     /// <summary>
     /// MSBuild property names that are required to be threaded as analyzer config options.
     /// </summary>
-    internal static partial class MSBuildPropertyOptionNames
+    internal static class MSBuildPropertyOptionNames
     {
-        public const string TargetFramework = "TargetFramework";
-        public const string TargetPlatformMinVersion = "TargetPlatformMinVersion";
+        public const string TargetFramework = nameof(TargetFramework);
```

Comment:
@sharwell I explicitly avoided using nameof here so this does not break if someone decides to rename this field to say, `TargetFrameworkPropertyName`. I would be much more comfortable with this change after @buyaa-n adds analyzer functionality that consumes this property + unit tests guarding it.

Reply:
Ok, i will add a unit test